src/lib.rs

```diff
@@ -142,8 +145,10 @@ impl FragmentScanner {
                 };
                 if !meta.file_type().is_file() {
                     if let Ok(target) = fs::read_link(&fpath) {
+                        let target = target.as_path();
+                        let devnull = std::path::Path::new(DEVNULL);
                         // A devnull symlink is a special case to ignore previous file-names.
-                        if target == path::PathBuf::from("/dev/null") {
+                        if target == devnull {
```

Comment:
this is a weird spelling; Path::new is 100% free, just make it `target == Path::new("/dev/null")`

Reply:
Yep, agreed; though mind reviewing https://github.com/coreos/liboverdrop-rs/pull/18 so I can just type `Path` here as you suggest instead of `std::path::Path`, then I'll rebase this on top.

Reply:
done